riptide-spring-boot-starter/src/main/java/org/zalando/riptide/spring/testing/RiptideClientTest.java

```diff
@@ -0,0 +1,41 @@
+package org.zalando.riptide.spring.testing;
+
+import org.springframework.boot.autoconfigure.ImportAutoConfiguration;
+import org.springframework.boot.test.autoconfigure.web.client.AutoConfigureMockRestServiceServer;
+import org.springframework.boot.test.autoconfigure.web.client.RestClientTest;
+import org.springframework.core.annotation.AliasFor;
+import org.springframework.test.context.TestPropertySource;
+import org.zalando.riptide.spring.RiptideAutoConfiguration;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Inherited;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Target(ElementType.TYPE)
+@Retention(RetentionPolicy.RUNTIME)
+@Documented
+@Inherited
+@RestClientTest
+@AutoConfigureMockRestServiceServer(enabled = false) // see RiptideClientAutoConfiguration
+@TestPropertySource(properties = {
+        "riptide.mocked: true"
```

Comment:
There it is.

Reply:
Also referencing your comment above: any other method to inject the "mock mode" is obviously fine - this has been just the fastest way :grin: